lib/attr_encrypted_pgcrypto/log_subscriber/postgres_pgp.rb

```diff
@@ -9,7 +9,8 @@ module PostgresPgp
       extend ActiveSupport::Concern
 
       included do
-        alias_method_chain :sql, :postgres_pgp
+        alias_method :sql_without_postgres_pgp, :sql
+        alias_method :sql, :sql_with_postgres_pgp
```

Comment:
Module#prepend could be an option here too

Reply:
Great suggestion, thanks! That cleaned up the code substantially.